resources/dicts/patrols/new_cat.json

```diff
@@ -296,8 +296,8 @@
     "win_trait": ["careful", "insecure", "nervous"],
     "min_cats": 1,
     "max_cats": 1,
-    "antagonize_text": null,
-    "antagonize_fail_text": null,
+    "antagonize_text": "r_c can sense it's a trap, and waits just outside, setting their own trap. They manage to set up such a yowl that it sends the rogues running as if LionClan were after them.",
+    "antagonize_fail_text": "r_c can sense its a trap, and tries to get the upper paw, only to find rogues waiting for them there too. r_c hopes they'll make it to StarClan",
```

Comment:
I'm fairly certain antagonize fails can't kill the cat atm? unless i've missed a new development lol